### PR TITLE
Fix ErfModel

### PR DESCRIPTION
**Pull request by cosama**
Closes https://github.com/lbl-anp/becquerel/issues/305.

@jvavrek Check this out if it works for you. If you have a good test for this, based on the case where you run into this, it would probably be a good idea to add this.

### CODE REVIEW

File: becquerel/core/fitting.py [hunk 1]
```diff
@@ -201,7 +201,7 @@ def guess(self, y, x=None, dx=None, center_ratio=0.5, width_ratio=0.5):
         msk = (x >= (mu - xspan * width_ratio)) & (x <= mu + xspan * width_ratio)
 
         # TODO: update this, minimizer creates NaN's if default sigma used (0)
```
Comment (jvavrek) [inline becquerel/core/fitting.py:203]: @cosama can you just remove this an related TODOs?

**Review by jvavrek — APPROVED** [review]
Looks good to me. Just update any out-of-date TODOs about guess values and good to go!

**Comment by markbandstra** [comment]
Didn't notice this at the time but we should do better on our commit messages -- a2aeb1e is "bump" and seems to be entirely empty (how did that happen?), and d43ce2f is "cosama" and a single-line change.

**Comment by jvavrek** [comment]
> Didn't notice this at the time but we should do better on our commit messages -- [a2aeb1e](https://github.com/lbl-anp/becquerel/commit/a2aeb1ebd78452e5d543171556c891c44a5c7936) is "bump" and seems to be entirely empty (how did that happen?), and [d43ce2f](https://github.com/lbl-anp/becquerel/commit/d43ce2f5525f8e1762ae7f2a5d1e504b17eef20d) is "cosama" and a single-line change.

Those were just test commits to try re-triggering the frozen CI (though I suppose the messages could have said that, yes)

**Comment by markbandstra** [comment]
Got it, thanks!

**Pull request merged**